app/models/ability.rb

```diff
@@ -94,6 +94,7 @@ def signed_in(user)
   # Abilities for users with roles wandering around in non-admin views.
   def common_abilities_for_admins(user)
     can :access, Admin
+    can :create, TicketScanning
```

Comment:
this should be moved to like 105 and 125 (to allow only organizers and info desk people to register people) and using the id of the conference :wink: